controller.go

```diff
@@ -106,14 +119,16 @@ type driverData struct {
 type driverTable map[string]*driverData
 type networkTable map[types.UUID]*network
 type endpointTable map[types.UUID]*endpoint
-type sandboxTable map[string]*sandboxData
+type containerTable map[string]*sandboxData
```

Comment:
`controller.sandboxes` contains the OS sandboxes, collection of `sandbox.Sandbox` objects.
`controller.containers` contains the collection of CNM Sandboxes.

Too many sandboxes...